Makefile

```diff
@@ -1,66 +1,151 @@
 # The main makefile to run all experiments in different configurations.
 #
-# Igor Konnov, 2020
+# Igor Konnov, Shon Feder, 2020
+
+
+##################
+# BASIC ELEMENTS #
+##################
+
+# These are the fundamental elements determining what benchmarks are run
+
+# The versions of apalache to benchmark
+#
+# A version should correspond to a docker tag for an image available on
+# https://hub.docker.com/r/apalache/mc/tags
+#
+# Put the latest version first
+VERSIONS := \
+0.7.0 \
+0.6.0
```

Comment:
should we have 0.6.1 here? That was a bugfix version.

Reply:
Fixed. Thank you!